index.bs

```diff
@@ -2899,14 +2898,14 @@ parameter to the {{CredentialsContainer/create()}} or {{CredentialsContainer/get
 
 <div dfn-type="dict-member" dfn-for="PublicKeyCredentialDescriptor">
     :   <dfn>type</dfn>
-    ::  This member contains the type of the [=public key credential=] the caller is referring to.
+    ::  This member contains the type of the [=public key credential=] the caller is referring to. The value SHOULD be a member of {{PublicKeyCredentialType}} but [=client platforms=] MUST ignore unknown values, treating an unknown value as if the member was unset.
 
     :   <dfn>id</dfn>
     ::  This member contains the [=credential ID=] of the [=public key credential=] the caller is referring to.
 
     :   <dfn>transports</dfn>
     ::  This OPTIONAL member contains a hint as to how the [=client=] might communicate with the [=managing authenticator=] of the
-        [=public key credential=] the caller is referring to. The values SHOULD be members of {{AuthenticatorTransport}} but [=client platforms=] MUST ignore unknown values.
+        [=public key credential=] the caller is referring to. The values SHOULD be members of {{AuthenticatorTransport}} but [=client platforms=] MUST ignore unknown values, treating an unknown value as if the member was unset.
```

Comment:
I think just leaving it at &ldquo;ignore unknown values&rdquo; is fine? Since it's a list, we just want to filter unknown values rather than discard the whole list.

Reply:
Hmm, I thought it was a bit ambiguous as it was, but I'll revert it. Thanks!